package.json

```diff
@@ -20,7 +20,10 @@
     "prop-types": "^15.7.2",
     "react": "17.0.1",
     "react-dom": "17.0.1",
-    "sass": "^1.30.0"
+    "react-hook-form": "^6.15.5",
+    "react-modal": "^3.12.1",
```

Comment:
is this really needed? its very easy for us to create our own `Modal` component using html and css.
I can help you if you are getting stuck. Let's implement it rather than using a package for this.